lib/requests/meal_requests.js

```diff
@@ -0,0 +1,78 @@
+//wrap this whole thing in a class (for ref: http://backend.turing.io/module4/lessons/intro-to-oojs)
+//create a new file and class that contains EventHandler, then call those things in here
+
+const API = "http://localhost:3000"
+var deleteImageSrc = 'https://assets.publishing.service.gov.uk/media/55b9f41b40f0b6151f000019/sign-giving-order-no-entry-vehicular-traffic.jpg'
+
+
+$(document).ready(function(){
+
+
+
+//shows all foods
+  fetch('http://localhost:3000/api/v1/meals')
+  .then( function(response) {
+      response.json().then(function(data) {
+        for(var i = 0; i < data.length; i++){
+          $(`#${data[i].name}-table`).append(
+            data[i].foods.forEach(function (food, index){
+               $(`#${data[i].name}-table`).append('<tr><td class="'+ food.name +'">' + food.name + '</td><td class="' + data[i].name + index + "cals" + '">' + food.calories + '</td><td class="deletor"><img class="del-image" src="' + deleteImageSrc + '"/></td></tr>');
```

Comment:
I know we talked about this, just a reminder for later:
var tableRow = document.createElement('tr');
    $(tableRow).addClass('food-all')
    $(tableRow).append('<td class="food ' +data[position]['id'] + '">' + data[position]['name'] + '</td>');
    $(tableRow).append('<td>' +data[position]['calories'] + '</td>');